Makefile

```diff
@@ -47,38 +43,38 @@ default: help
 .PHONY: docker/build
 ## Build the docker image
 docker/build:
-	@docker build -t ${DOCKER_HUB_REPO}:latest -t ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} .
+	docker build -t ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} .
 
 .PHONY: docker/tag
-## Create a new tag ( expects the "DOCKER_IMAGE_ADDITIONAL_TAG" environment variable to be set )
+## Create a new tag
 docker/tag:
-	@docker tag ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_ADDITIONAL_TAG}
+	docker tag ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} ${DOCKER_HUB_REPO}:latest
 
 .PHONY: docker/save
 ## Save the docker image to disk
 docker/save:
-	@docker save ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} > "${DOCKER_IMAGE_TAG}.tar"
+	docker save ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} > "docker-image.tar"
```

Comment:
why? it's a temporary file... which is used inside and outside of the makefile... so it should be known by both worlds.... it's a bad situation anyway...

Reply:
yeah but the file reflects a version of the docker image. If you work in a local environment, where you always work with latest ( no version specified), this will just cause confusion .. we should stay explicit and expressie.